src/main.cpp

```diff
@@ -2574,11 +2582,7 @@ void static UpdateTip(CBlockIndex *pindexNew) {
             ThresholdState state = checker.GetStateFor(pindex, chainParams.GetConsensus(), warningcache[bit]);
             if (state == THRESHOLD_ACTIVE || state == THRESHOLD_LOCKED_IN) {
                 if (state == THRESHOLD_ACTIVE) {
-                    strMiscWarning = strprintf(_("Warning: unknown new rules activated (versionbit %i)"), bit);
-                    if (!fWarned) {
-                        AlertNotify(strMiscWarning, true);
-                        fWarned = true;
-                    }
+                    AlertNotifyOnce(strprintf(_("Warning: unknown new rules activated (versionbit %i)"), bit));
```

Comment:
Reading this again, I think this one should not be `AlertNotifyOnce`, but `AlertNotify`. We can have several parallel version bits activation notified...